pkgs/applications/version-management/git-and-tools/hub/default.nix

```
@@ -13,7 +13,7 @@ buildGoPackage rec {
     owner = "github";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0yxpr606xx23l8823hjqj16cvjjrwb28c7z08ml1pkfvaf7w4n81";
+    sha256 = "1vvrc3k81jm9c664g0j9666i7ypn7n7jfyj4gxcybq3sg2d4di27";
   };
 
   nativeBuildInputs = [ groff ronn utillinux glibcLocales ];
```

```suggestion
  nativeBuildInputs = [ groff ronn utillinux ];
```

Don't think we need it anymore.
Also need to remove the `LC_ALL=en_US.utf-8` bit below and use C.UTF-8